pkg/cmd/version/version.go

```diff
@@ -0,0 +1,25 @@
+package version
+
+import (
+	"fmt"
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdVersion(f factory.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:    "version",
+		Hidden: true,
```

Comment:
Github CLI had the 'version' command as hidden, which seemed reasonable so I copied here